draft-group-privacypass-k-check.md

```diff
@@ -274,9 +274,10 @@ following:
 }
 ~~~
 
-Clients take each “token-key” parameter, base64url-decode the result, and then
-derive a token key identifier as defined in {{Section 6.5 of PRIVACYPASS-ISSUANCE}}.
-Each token key identifier is a normalized representation of the resource.
+Clients compute the first valid representation of this directory, which might be the key ID
+of the first key in the "token-keys" list (depending on the "not-before" value), or the
+key ID of the second key in the "token-keys" list. The key ID is computed as defined in
+{{Section 6.5 of PRIVACYPASS-ISSUANCE}}.
```

Comment:
Is this saying that the "first valid representation" is:
1. The first resource, sorted by "not-before" value, if present
1. Otherwise, the first resource, sorted lexicographically by keyid?

Reply:
It's saying that it's the first key in the list that the client would attempt to use.

Reply:
Might be nice to clarify "the first valid representation (the first entry in the list that the client is able to use)"...